lambdas/generator/handler.py

```diff
@@ -103,7 +86,7 @@ def get_programming_languages_used(github_username, repository_name):
     return languages
 
 
-def get_important_file_names(github_username, repository_name, languages):
+def get_important_file_names(languages):
```

Comment:
Same as below. Would be helpful to use a different word other than 'important' and add a description with expected output